### ISSUE TITLE
2 missing classes?

### ISSUE THREAD

**Issue by larshp** [hardyp/ABAPToTheFuture04#4]
`zcx_violated_precondition` and `zcx_violated_postcondition` seems to be missing?

**Comment by hardyp** [comment]
My understanding - and it could be wrong - is that this will be magically handled by APACK. I created a class in the main package which had the marker interface ZIF_APACK_MANIFEST. I then filled the CONSTRUCTOR as follows.

![image](https://user-images.githubusercontent.com/7078215/142143706-23116185-9a0a-41e6-a4c0-6ec2d5701e2e.png)

How it was sold to me was that somehow when someone downloads the repository the APACK class is read and then something looks to see if the dependencies are already installed and if not downloads them from the relevant GitHub repository. Somewhat like SNOTE which makes sure OSS notes are downloaded in the correct order.
